pkg/util/command_util.go

```diff
@@ -352,7 +352,7 @@ Loop:
 
 func GetUserGroup(chownStr string, env []string) (int64, int64, error) {
 	if chownStr == "" {
-		return DoNotChangeUID, DoNotChangeGID, nil
+		return 0, 0, nil
```

Comment:
Hi @KerchumA222 this default change looks good.
For the nit, would you mind also following the previous convention for naming it as DefaultUID?
Thanks!

Reply:
@JeromeJu I updated the code. Is this what you meant?

Reply:
Thanks @KerchumA222